frontend/packages/console-dynamic-plugin-sdk/src/app/useReduxStore.ts

```diff
@@ -22,7 +22,7 @@ const composeEnhancers =
  * }
  * ```
  */
-export const useReduxStore = (): { store: Store<SDKStoreState>; storeContextPresent: boolean } => {
+export const useReduxStore = (): { store: Store<any>; storeContextPresent: boolean } => {
```

Comment:
Undo?

Reply:
type of redux store can depend on host app as well like if they have a store already in the tree i.e https://github.com/openshift/console/blob/master/frontend/public/redux.ts#L18-L26. So changed it to any from `SDKStoreState`.

Reply:
It's polymorphic... make it generic if you have to, but `any` isn't right.